pkg/minikube/cluster/cluster.go

```diff
@@ -224,6 +225,12 @@ func createDriverOptions(driver drivers.Driver, explicitOptions map[string]inter
 
 func (m *MachineConfig) CacheMinikubeISOFromURL() error {
 	fmt.Println(fmt.Sprintf("Downloading ISO '%s'", m.MinikubeISO))
+	tmpISOFile, err := ioutil.TempFile("", "tmpISOFile")
+	fmt.Println("TMP FILE IS", tmpISOFile.Name())
+	if err != nil {
+		return err
+	}
+	defer os.RemoveAll(tmpISOFile.Name())
 
 	// store the iso inside the MINISHIFT_HOME dir
```

Comment:
I think this comment is a bit misleading here. Maybe instead of this comment we should add a proper GoDoc description on what this method does.